site/docs/voice/about.mdx

```diff
@@ -33,14 +33,22 @@ The Bandwidth Voice API allows you to handle incoming and outgoing PSTN phone ca
 
 `https://voice.bandwidth.com/api/v2/accounts/{accountId}`
 
+**Note** HTTP requests to our API use HTTP Basic authentication, which refers to your account's username and password.
+
 ## Calls
 
+For help on how to make the proper requests to the API, check out the [API Reference](/apis/voice#operation/createCall) section of the docs. To quickly start making and receiving calls, check out the [guide](/docs/voice/quickStart).
+
 ### Creation/Acceptance
 
-A <Highlight color="#079CEE">POST</Highlight> to `/calls` creates a call object in Bandwidth's network and returns a promise in the form of a unuqie `callId`. When creating a call you provide an `answerUrl` in the request body, and upon answer Bandwidth will send a webhook to the provided URL confirming and expecting BXML in the HTTP response to continue with the call.
+A <Highlight color="#079CEE">POST</Highlight> to `/calls` creates a call object in Bandwidth's network and returns a promise in the form of a unique `callId`. When creating a call you provide an `answerUrl` in the request body, and upon answer Bandwidth will send a webhook to the provided URL confirming and expecting BXML in the HTTP response to continue with the call.
```

Comment:
`promise`? That seems like odd phrasing to me since we're not talking about JavaScript.

Reply:
```suggestion
A <Highlight color="#079CEE">POST</Highlight> to `/calls` creates a call object in Bandwidth's network and returns a promise in the form of a unique `callId`. When creating a call you provide an `answerUrl` in the request body, and upon the call being answered, Bandwidth will send a webhook to the provided URL to inform your application and retrieve BXML from it in order to continue with the call.
```